assets/stylesheets/utilities/_utilities.scss

```diff
@@ -709,11 +710,19 @@ a.underline,
   width: 100%;
 }
 
+.full-height {
+  height: 100%;
+}
+
+.flex {
```

Comment:
I'm not sure how I feel about creating this as a helper class. Flex requires so many additional rules to create a layout that I'm having a hard time seeing a benefit here.

And it might just be me, but using flex with Bootstrap for component layouts is confusing. Bootstrap uses floats to manage its layout and floats don't work with flexbox. It's like oil and water essentially - it makes more sense using one.